### PR TITLE
JAMES-2234 Document ASCII mail address restriction in WebAdmin

### PR DESCRIPTION
**Pull request by chibenwa**

### CODE REVIEW

File: src/site/markdown/server/manage-webadmin.md [hunk 1]
```diff
@@ -435,6 +435,8 @@ This feature uses [Recipients rewrite table](/server/config-recipientrewritetabl
 the [RecipientRewriteTable mailet](https://github.com/apache/james-project/blob/master/server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/RecipientRewriteTable.java)
 to be configured.
 
+Note that email addresses needs to be composed of ASCII only characters. Mail addresses not matching this criteria will be rejected.
```
Comment (quynhn) [inline src/site/markdown/server/manage-webadmin.md]: more space after '+'?

Comment (chibenwa) [reply src/site/markdown/server/manage-webadmin.md]: '+' is the marker for addition in Git. 

It's actually note contained in my proposal.

Comment (strokyl) [reply src/site/markdown/server/manage-webadmin.md]: lol

Comment (strokyl) [reply src/site/markdown/server/manage-webadmin.md]: There is more than one addresses so s/needs/need/.
Also I think "ASCII only characters" is weird. "only ASCII characters" sound better too me

**Comment by rouazana** [comment]
no need to document it in swagger?

File: src/site/markdown/server/manage-webadmin.md [hunk 2]
```diff
@@ -435,7 +435,7 @@ This feature uses [Recipients rewrite table](/server/config-recipientrewritetabl
 the [RecipientRewriteTable mailet](https://github.com/apache/james-project/blob/master/server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/RecipientRewriteTable.java)
 to be configured.
 
-Note that email addresses needs to be composed of ASCII only characters. Mail addresses not matching this criteria will be rejected.
+Note that email addresses needs to be composed of only ASCII characters. Mail addresses not matching this criteria will be rejected.
```
Comment (strokyl) [inline src/site/markdown/server/manage-webadmin.md]: needs still misspelled

File: server/protocols/webadmin/webadmin-data/src/main/java/org/apache/james/webadmin/routes/GroupsRoutes.java [hunk 3]
```diff
@@ -83,6 +83,8 @@
     private static final String GROUP_ADDRESS_PATH = ROOT_PATH + SEPARATOR + ":" + GROUP_ADDRESS;
     private static final String USER_ADDRESS = "userAddress";
     private static final String USER_IN_GROUP_ADDRESS_PATH = GROUP_ADDRESS_PATH + SEPARATOR + ":" + USER_ADDRESS;
+    private static final String MAILADDRESS_ASCII_DISCLAIMER = "Note that email addresses needs to be composed of only " +
```
Comment (strokyl) [inline server/protocols/webadmin/webadmin-data/src/main/java/org/apache/james/webadmin/routes/GroupsRoutes.java]: s/needs/need/

File: server/protocols/webadmin/webadmin-data/src/main/java/org/apache/james/webadmin/routes/GroupsRoutes.java [hunk 4]
```diff
@@ -130,8 +132,13 @@ public void define(Service service) {
     @Path(ROOT_PATH + "/{" + GROUP_ADDRESS + "}/{" + USER_ADDRESS + "}")
     @ApiOperation(value = "adding a member into a group")
     @ApiImplicitParams({
-        @ApiImplicitParam(required = true, dataType = "string", name = GROUP_ADDRESS, paramType = "path"),
-        @ApiImplicitParam(required = true, dataType = "string", name = USER_ADDRESS, paramType = "path")
+        @ApiImplicitParam(required = true, dataType = "string", name = GROUP_ADDRESS, paramType = "path",
+            value = "Mail address of the groups. Sending a mail to that address will send it to all group members.\n" +
+            MAILADDRESS_ASCII_DISCLAIMER),
+        @ApiImplicitParam(required = true, dataType = "string", name = USER_ADDRESS, paramType = "path",
+            value = "Mail address of the group. Sending a mail to the group mail address will send an email to " +
```
Comment (strokyl) [inline server/protocols/webadmin/webadmin-data/src/main/java/org/apache/james/webadmin/routes/GroupsRoutes.java]: group or groups ? Be consistant with above

**Comment by chibenwa** [comment]
test this please

**Pull request closed without merging**